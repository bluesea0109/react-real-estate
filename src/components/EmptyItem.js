import React from 'react';

import { Header, Icon, Segment } from './Base';

const EmptyItem = () => {
  return (
    <Segment placeholder>
      <Header icon>
        <Icon name="file outline" />
        No Mailouts found.
      </Header>
    </Segment>
  );
};

export default EmptyItem;
