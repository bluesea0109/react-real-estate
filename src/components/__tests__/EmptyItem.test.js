import React from 'react';
import renderer from 'react-test-renderer';
import EmptyItem from '../EmptyItem';
import 'jest-styled-components';

it('renders correctly', () => {
  const tree = renderer.create(<EmptyItem />).toJSON();
  expect(tree).toMatchSnapshot();
});
