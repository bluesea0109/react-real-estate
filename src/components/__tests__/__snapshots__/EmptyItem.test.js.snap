// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="ui placeholder segment "
>
  <div
    className="ui icon header "
  >
    <i
      aria-hidden="true"
      className="file outline icon "
      onClick={[Function]}
    />
    No Mailouts found.
  </div>
</div>
`;
